cime_config/cesm/machines/Makefile

```diff
@@ -776,8 +770,8 @@ cleanrof:
 	cd $(EXEROOT)/rof/obj ; $(RM) -f *.o *.mod
 
 cleanlnd:
-	$(RM) -f $(LIBROOT)/liblnd.a
-	cd $(EXEROOT)/lnd/obj ; $(RM) -f *.o *.mod
+	$(RM) -f $(LNDLIBDIR)/$(LNDLIB)
```

Comment:
I have tested the yellowstone prealpha intel test suite which does include several I cases.   Is there a suite that tests all three flavors of clm?   Is there a baseline I can compare to?